src/Snippets/Docs/Telemetry.cs

```diff
@@ -147,15 +147,15 @@ public static void SeverityOverrides()
             // This ensures that common configuration is preserved.
             var telemetryOptions = new TelemetryOptions(context.GetOptions<TelemetryOptions>());
 
-            telemetryOptions.SeverityProvider = ev =>
+            telemetryOptions.SeverityProvider = @event =>
```

Comment:
I guess now this would be better named `args`?

Reply:
Done.